resource-definitions/crd_injector.go

```diff
@@ -21,41 +22,50 @@ var (
 	CRDName   = "configurations." + GroupName
 )
 
-func getHomeDir() string {
-	if home := os.Getenv("HOME"); home != "" {
-		return home
-	}
-	return os.Getenv("USERPROFILE")
+var crdSchema = &apiextensionsv1.JSONSchemaProps{
+	// NOTE: Take the CRD schema from the internal/configurationmanagers/kubernetes/crd/crd_injector.go
 }
 
-func createCRD(apiExtClient *apiextensionsclientset.Clientset) {
-	secretlessCRD := &apiextensionsv1beta1.CustomResourceDefinition{
-		ObjectMeta: meta_v1.ObjectMeta{
-			Name: CRDName,
-		},
-		Spec: apiextensionsv1beta1.CustomResourceDefinitionSpec{
-			Group: GroupName,
-			Names: apiextensionsv1beta1.CustomResourceDefinitionNames{
-				Kind:   "Configuration",
-				Plural: "configurations",
-				ShortNames: []string{
-					"sbconfig",
-				},
+// This schema should be updated to match secretless-resource-definition.yaml
```

Comment:
Same comment as earlier... maybe this should be "This schema should be kept up-to-date to match..." or something similar?